### ISSUE TITLE
Inventarsystem dokumentieren

### ISSUE THREAD

**Issue by Profpatsch** [openlab-aux/orgafoo#215]
Bitte eine Seite auf dem [Wiki](http://wiki.openlab-augsburg.de/openwiki:start) anlegen, und das Backend und Frontend ausführlich dokumentieren und unter `Infrastruktur` eintragen.

cc @glaxx @waaaaargh

**Comment by pRiVi** [comment]
Vor allem die Orte wo das Zeugs liegt, und wie man da hin kommt wäre wichtig.

**Comment by glaxx** [comment]
@Profpatsch Was stellst du dir da vor?

Der Source Code vom Backend ist öffentlich [auf Github](https://github.com/openlab-aux/lsmsd) verfügbar.

**Comment by Profpatsch** [comment]
@glaxx Naja, Funktionalität und wie man es benutzt. Und wie man es zum Laufen bringt, wenn es nicht funktioniert.

**Comment by lukasepple** [comment]
After all it's called RTFM, not RTF Source Code.

**Comment by pRiVi** [comment]
@glaxx: Mich würde insbesondere interessieren wo es gespeichert ist und wie man da per SSH oder ähnlichem hin kommt.

**Comment by glaxx** [comment]
@Profpatsch @lukasepple https://github.com/openlab-aux/lsmsd#installation
https://github.com/openlab-aux/lsmsd#rest-api-documentation

@pRiVi Das weiß nur der @waaaaargh .

**Comment by lukasepple** [comment]
@glaxx dann verlinkt das doch im Wiki :)

@waaaaargh Frickelclient sollte man auch dokumentieren oder gleich eine ordentlich GUI schreiben.

**Comment by lukasepple** [comment]
War nicht @glaxx der Qt-Wizard? :runner:

**Comment by salomonderossi** [comment]
@lukasepple Ich hab damals(tm) angefangen einen Client in python + qt zu schreiben, aber aus Zeitgründen nicht zu Ende gebracht. Dann hat Hannes den Frickelclient geschrieben und dann hab ich das endgültig Ad acta gelegt...
Hier zu finden: https://github.com/openlab-aux/hamster

Ich würde mich freuen, wenn sich Jemand findet, der das übernehmen kann. Mir geht leider die Zeit aus.

**Comment by waaaaargh** [comment]
@pRiVi Inventarsystem-Deployment liegt auf einer Rantanplan-VM, die ich aufgesetzt habe. Bei Bedarf kann ich Dir einen SSH-Key auf der Kiste hinterlegen. Fährt mein standardmäßiges Setup mit supervisord.

**Comment by Profpatsch** [comment]
Jo, im Grunde sollte auf dem Wiki die Instanz im Lab dokumentiert werden. Und zwar am besten so, dass es trivial reproduzierbar ist.

**Comment by sternenseemann** [comment]
Gerne hier dokumentieren <http://openlab-augsburg.de:5001/Inventarsystem>

**Comment by glaxx** [comment]
Ihr wisst inzwischen alle ueber meine zukuenftigen Plaene bescheid. Wie ist denn der **aktuelle** Stand, was wuerdet **Ihr** euch noch an Dokumentation wuenschen?

**Comment by richi235** [comment]
> Ihr wisst inzwischen alle ueber meine zukuenftigen Plaene bescheid.

Nope, ich habe keine Ahnung was du in 5 Jahren für Pläne hast. Ich kenne ja vermutlich auch nur einen kleinen Teil deiner gegenwärtigen Pläne. Worum genau geht es denn konkret?

**Comment by Profpatsch** [comment]
lel, Inventarsystem.
